logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/EndCallUseCase.kt

```diff
@@ -8,10 +9,12 @@ import kotlinx.coroutines.withContext
 
 class EndCallUseCase(
     private val callManager: Lazy<CallManager>,
+    private val callRepository: CallRepository,
     private val dispatchers: KaliumDispatcher = KaliumDispatcherImpl
 ) {
 
     suspend operator fun invoke(conversationId: ConversationId) = withContext(dispatchers.io) {
         callManager.value.endCall(conversationId)
+        callRepository.updateIsCameraOnById(conversationId.toString(), false)
```

Comment:
What if the call is not actively ended by the user but closed due to connectivity issues? Should we call `updateIsCameraOnById()` in those cases as well?

Reply:
No in case of timeout we don't need to call `updateIsCameraOnById()` because the call is `CLOSED`(which is no longer useful ). We just need it for calls that are still ongoing